docs/source/quick_start.rst

```diff
@@ -0,0 +1,103 @@
+Quick Start
+============
+
+This guide is intended to jump start new Kokkos users (and beginners, in particular).
+
+
+Download Latest and Build 
+-----------------------------
+
+.. note::
+
+  Please become familiar with `Kokkos Requirements <https://kokkos.org/kokkos-core-wiki/requirements.html>`_, and verify that your machine has all necessary compilers, backend GPU SDK (e.g., CUDA, ROCM, Intel oneAPI, etc.),and build system components.
+
+
+..
+ Nota bene:  the link for "Latest" should be stable from one release to the next, but check periodically to be sure 
+..
+
+:bdg-link-primary:`Latest Release <https://github.com/kokkos/kokkos/releases/latest>`
+
+.. code-block:: sh
+  
+  # Uncomment according to the type of file you've downloaded (zip or tar)
+  unzip kokkos-x.y.z.zip 
+  # tar -xzf kokkos-x.y.z.tar.gz
+  cd kokkos-x.y.z
+
+
+Basic Configuration
+-------------------
+
+.. note::
+
+  You can create small shell scripts to manage and experiment with configuration details, following the GPU microarchitecture-appropriate examples below.  Upon successful configuration, ``make `` to build, and ``make install`` to install.
+
+
+
+OpenMP (CPU Parallelism)
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. code-block:: sh
+
+  cmake -B <name-of-build-directory> -DKokkos_ENABLE_OPENMP=ON -DCMAKE_BUILD_TYPE=Release -S <name-of-source-directory>
+
+
+.. note::
+
+  Kokkos will attempt to autodetect GPU microarchitecture, but it is also possible to specify the desired `GPU architecture <https://kokkos.org/kokkos-core-wiki/keywords.html#gpu-architectures>`_.  In scenarios where a backend (e.g., CUDA, HIP) is enabled, Kokkos will execute serially on the "host" (CPU).  
```

Comment:
There is a slight bug in the wording here, it implies that if a backend is enabled Kokkos will run serially. We really mean that if a device backend is enabled, the default on the host will be serial

```suggestion
  Kokkos will attempt to autodetect GPU microarchitecture, but it is also possible to specify the desired `GPU architecture <https://kokkos.org/kokkos-core-wiki/keywords.html#gpu-architectures>`_.  In scenarios where a device (GPU) backend (e.g., CUDA, HIP) is enabled, Kokkos will default to serial execution on the host (CPU).
```

Reply:
you can keep the quotes if you prefer but if you add them back add to "device" also